Encryption.md

```diff
@@ -138,13 +138,13 @@ union EncryptionAlgorithm {
 ```
 
 The `AesGcmV1` and `AesGcmCtrV1` structures contain an optional `aad_metadata` field that can 
-be used by a reader to retrieve the AAD string used for file encryption. The maximal allowed
+be used by a reader to retrieve the file AAD string used for file enciphering. The maximal allowed
 length of `aad_metadata` is 512 bytes. Typically, readers would know the file AAD without relying
 on this field, using instead a method adopted by the organization. For example, the AAD can be the name of 
 the file - if constructed with a timestamp or version number, the AAD will allow readers to make sure they
 work with a correct file version. If organization works with a number of different 
-AAD construction methods, the `aad_metadata` can contain the method ID. In any case, the AAD string itself
-must not be kept in this field - a reader reliance on this AAD will defy the cipher authentication purpose.
+AAD construction methods, the `aad_metadata` can contain the method ID; otherwise, the `aad_metadata` can
```

Comment:
Why did you remove the sentence that the AAD itself must not be kept in this field? I think that was one of the most important points. In fact, I think you should elaborate on that and describe the intended use of this field. If I understand correctly: if there are multiple AADs in use, we need a way to decide which one to use in each case. Storing some kind of ID or reference to the AAD in this field solves this purpose. Is this the intended use case? Is there any other one? Please describe them here briefly.

Reply:
Turns out there is a usecase where it might make sense to store the AAD in this field. Say we have a dataset with 3 files: customers_ny_oct18, customers_pa_oct18 and customers_ct_oct18. We can store these strings as AADs in the 'aad_metadata' field - as long as the reader knows it must get Oct18 version and checks for "customer_??_oct18" AADs. The main message is in the last part - a reader must know what dataset it needs, and what AADs should be there. I'm adding a section on AADs to the doc.